firefox_media_tests/playback/test_video_playback.py

```diff
@@ -57,6 +57,6 @@ def run_playback(self, timeout=0):
                     # We want to make sure that the current time is near the
                     # duration we want.
                     if video.current_time < (start_time + duration):
```

Comment:
Aside: It would be safer to check that `current_time` is within some threshold of `start_time + duration`, e.g. half a second.